book.js

```diff
@@ -31,6 +31,19 @@ class bookStore
        console.log(books);
        localStorage.setItem('books',JSON.stringify(books));
    }
+   static clearBook()
+   {   let books=bookStore.getBooks();
+          books=[];
+      localStorage.setItem('books',JSON.stringify(books));
+      
+     let n=document.querySelectorAll('tr').length;
```

Comment:
can you explain this function. What it is doing?

Reply:
> Can you explain this code, so that I can do the necessary changes
> ![book js - KnowledgeShare - Visual Studio Code 12-12-2020 23_45_07](https://user-images.githubusercontent.com/68608356/101991703-6b8bf600-3cd4-11eb-833b-2c461ab438e1.png)

for deleting the list on UI

Reply:
> can you explain this function. What it is doing?

It is clearing the local storage.  Then in a loop, it will delete the list on UI